docs/source/variables.ipynb

```diff
@@ -225,13 +253,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This primitive can now be passed to `ft.dfs` as a `trans_primitives`. \n",
-    "\n",
-    "```python\n",
+    "This primitive can now be passed to `ft.dfs` as a `trans_primitives`. It will now generate a feature which uses our custom primitive (**AgeOver100**) with the custom variable type (**Age**)."
```

Comment:
> This primitive can now be passed to `ft.dfs` as a `trans_primitives`.

"as a `trans_primitives`" is a little awkward

> It will now generate

Instead of "It" maybe "DFS" or "`dfs`"

Reply:
Fixed both:
https://github.com/FeatureLabs/featuretools/pull/1066/commits/2a1912404d84f9e8f7e486f50c8e86ccc2f9d0d0#diff-788cca9c6a491b8eed11068ab47ed94cR256